app/config.py

```diff
@@ -38,17 +38,14 @@ def __init__(self):
 
 
 class ApiTestingConfig(ApiConfig):
-    DATABASE_USERNAME = os.getenv('DATABASE_USERNAME')
-    DATABASE_PASSWORD = os.getenv('DATABASE_PASSWORD')
+    DATABASE_USERNAME = os.getenv('DATABASE_USERNAME', 'postgres')
```

Comment:
are there any security issues here with using postgres/postgres as user/pass? probably not, right, as this is only as part of CI/CD testing?

Reply:
Nah